mne/viz/epochs.py

```diff
@@ -953,6 +970,29 @@ def _plot_traces(params):
         ax.set_yticklabels(labels, fontsize=12, color='black')
     else:
         ax.set_yticklabels(tick_list, fontsize=12)
+
+    if params['events'] is not None:  # vertical lines for events.
+        color = params['event_colors']
+        for ev_line in params['ev_lines']:
+            ax.lines.remove(ev_line)  # clear the view first
+        params['ev_lines'] = list()
+        t_zero = np.where(epochs.times == 0.)[0]  # idx of 0s
```

Comment:
the fact that epochs do not have to contain the zero point is why the check is for `== 0` and in the line below the case when zero is not present is dealt with.